randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -1657,7 +1657,7 @@ Extra - asset_id: collision_camera_021
   > Pickup (Power Bomb Tank)
       Any of the following:
           Use Spin Boost
-          Bomb and Morph Ball and Infinite Bomb Jump (Intermediate)
+          Infinite Bomb Jump (Advanced) and Lay Bomb
```

Comment:
Like with others, this isnt doable with Cross. Separate it out.

Reply:
Done